docs/tutorial/process-model.md

```diff
@@ -2,7 +2,9 @@
 
 Electron inherits its multi-process architecture from Chromium, which makes the framework
 architecturally very similar to a modern web browser. In this guide, we'll expound on
-the conceptual knowledge of Electron that we applied in the minimal quick start app.
+the conceptual knowledge of Electron that we applied in the minimal [quick start app][].
```

Comment:
Just making sure that the empty brackets syntax works, because I think I saw one `that looked [like this] as well.`

Reply:
It seems like both syntaxes actually work in GitHub's own Markdown renderer, which _should_ be safe for our purposes as well: https://github.com/electron/electron/blob/40d8d3c1048e78a869629f26e3622b36ffe9ae95/docs/tutorial/process-model.md